tests/BaGetter.Core.Tests/Services/PackageIndexingServiceInMemoryTests.cs

```diff
@@ -151,9 +156,6 @@ public async Task IndexIMAsync_WhenPrereleasePackageAlreadyExists_AndOverwritePr
         var stream = new MemoryStream();
         builder.Save(stream);
 
-        _storage.Setup(s => s.DeleteAsync(builder.Id, builder.Version, default)).Returns(Task.CompletedTask);
-        _storage.Setup(s => s.SavePackageContentAsync(It.Is<Package>(p => p.Id == builder.Id && p.Version.ToString() == builder.Version.ToString()), stream, It.IsAny<FileStream>(), default, default, default)).Returns(Task.CompletedTask);
-
```

Comment:
Why did you remove the tests for Delete and Save?

Reply:
[PackageIndexingServiceInMemoryTests.cs](https://github.com/bagetter/BaGetter/blob/main/tests/BaGetter.Core.Tests/Services/PackageIndexingServiceInMemoryTests.cs) is a clone of [PackageIndexingServiceTests.cs](https://github.com/bagetter/BaGetter/blob/main/tests/BaGetter.Core.Tests/Services/PackageIndexingServiceTests.cs) that is there  to test the auto deletion. `_storage` uses an actual implementation instead of mock classes and assertions. The implementation provides an in-memory database and lets me test the whole workflow from addition to auto-deletion - at the same time this makes it more difficult to keep these assertions.